app/models/ems_refresh.rb

```diff
@@ -57,7 +57,7 @@ def self.queue_refresh(target, id = nil)
   end
 
   def self.refresh(target, id = nil)
-    EmsRefresh.init_console if MiqEnvironment::Process.is_rails_console?
+    EmsRefresh.init_console if defined?(Rails::Console)
```

Comment:
Replaced MiqEnvironment::Process call with a Rails constant check (so we can delete MiqEnvironment::Process)